src/Okta.DNX.OAuth.ResourceServer/config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Is there a README or some other instructions to let them know they need to update the configuration here? And in general what the purpose of this project is, and how to get it up and running?